src/Commands/StravaClientCommand.php

```diff
@@ -6,7 +6,7 @@
 
 class StravaClientCommand extends Command
 {
-    public $signature = 'strava-client';
+    public $signature = 'strava-client:install';
 
     public $description = 'My command';
 
```

Comment:
Write out some general description especially since this is a placeholder, maybe some comments about what this command might end up being, so once I or someone else comes back there's an idea on where to pickup.